### ISSUE TITLE
Unable to see Emoji of any kind in client [Bug]

### ISSUE THREAD

**Issue by Irresplendence** [BetterDiscord/BetterDiscord#1749]
### Before Reporting

- [X] I found no existing issues matching my bug
- [X] My issue is not caused by a plugin or theme

### Describe the Bug

I've been unable to unearth any information pertaining to this particular behavior that I'm seeing. 

For whatever reason, I am unable to see _any_ emoji, whether they are _Custom, or Default._

I assumed that this may be due to a plugin or a theme, but after having disabled them, I still cannot see emoji. 

![Discord Bug](https://github.com/BetterDiscord/BetterDiscord/assets/38499543/692470ca-b724-4e49-8a4a-ebe5eb3a7d6a)

For the time being, I've been using the regular discord client, which shows all of the emojis. 

I've also tried experimenting with emoji names, trying different sizes, but I believe all of that became moot when the default pre-loaded emotes also stopped functioning. I also messed with the accessibility options, as I typically run discord in **compact** mode. All of the configurations I'd tried resulted in the same behavior. 
 
I also could not find another bug that described my issue on the github issues. 

Any help is greatly appreciated. 

### Expected Behavior

Generally speaking, I would expect that the emoji appears in chat. 

### Steps To Reproduce

Versioning information:

```md
## Discord Info
stable 294520 (9ce8cb4) Host 1.0.9147 x64 (47834) Windows 10 64-bit (10.0.19045)

## BetterDiscord
stable 1.10.1

### Plugins (0 Enabled):
- BDFDB
- BetterVolume
- Hide Channels
- HideDisabledEmojis
- InvisibleTyping
- ReadAllNotificationsButton
- ZeresPluginLibrary

### Themes (0 Enabled):
- Alexis Modern Discord Theme
- AMOLED-Cord
- Comfy
- GitHub Dark
- Mabinogi
- Malice Mizer 1
- NotAnotherAnimeTheme
- Ocean
- Solana
- Unity
- Zelk
```

1. Launch Better Discord
2. Enter a server or DM.
3. Select the Emoji picker.
4. Select an Emoji.
5. Hit "SEND" 
6. Observe there is no emoji displayed.

### Operating System

Windows

### Discord Version

Stable

### BetterDiscord Version

Core v1.10.1

**Comment by Irresplendence** [comment]
I could just start fresh / backup my plugins and themes, but the gif was produced with all of them disabled.

**Comment by Irresplendence** [comment]
Update: I ended up starting _completely_ fresh. 

Prior to today, I was using the base client for discord, and decided to move back to BD. When starting BD, I had some previous data from the last time I'd used it. (Maybe a year ago or something?) So, I ended up clearing EVERYTHING, plugins, themes, anything in the Custom CSS, and uninstalled discord.

Reinstalled them both, and my emojis are back. Not sure why I didn't just try this before.

Thank you for looking at this!
